DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/Utils/SnapshotTestCase.swift

```diff
@@ -46,6 +46,8 @@ internal class SnapshotTestCase: XCTestCase {
         function: StaticString = #function
     ) throws {
         show(fixture: fixture)
+        // Give time for the view to appear and lay out properly
+        wait(seconds: 0.2)
```

Comment:
I had to add this, otherwise the snapshot test suite is failing after adding a new test for the activity indicator recorder. This is a temporary workaround for a [known issue](https://datadoghq.atlassian.net/browse/RUM-5130?atlOrigin=eyJpIjoiZTI5ODQ2Y2IwMDY5NDdlNWFlNDFjNzNjMTQ0YWNiNGIiLCJwIjoiamlyYS1zbGFjay1pbnQifQ).